packages/hasura-auth-js/src/utils/helpers.ts

```diff
@@ -11,7 +11,13 @@ import { ClientStorage, ClientStorageType, Session } from './types'
 export const isBrowser = () => typeof window !== 'undefined'
 
 export const getSession = (context?: AuthContext): Session | null => {
-  if (!context || !context.accessToken.value || !context.refreshToken.value) return null
+  if (
+    !context ||
+    !context.accessToken.value ||
+    !context.refreshToken.value ||
+    !context.accessToken.expiresAt
+  )
```

Comment:
to improve readability, I'd suggest that we should use curly braces around the return statement